### ISSUE TITLE
[no-commented-out-code]: False positives

### ISSUE THREAD

**Issue by Glavin001** [Glavin001/tslint-clean-code#44]
![image](https://user-images.githubusercontent.com/1885333/36646905-656f9240-1a54-11e8-916a-1a1e0560e3c1.png)

```
// tslint:disable:no-reserved-keywords
export interface Option {
  title?: string;
  type: "string" | "boolean" | "integer" | "array";
  description: string;
  /**
   * Available since version
   */
  since: SemVer;
  /**
   * Deprecated since version
   */
  deprecated?: SemVer;
  default?: any;
  enum?: any[];
  maximum?: number;
  minimum?: number;
  multipleOf?: number;
  items?: {
    type: string;
  };
}
// tslint:enable:no-reserved-keywords
```

![image](https://user-images.githubusercontent.com/1885333/36646909-6fd1e832-1a54-11e8-96dc-f5c8576fdf04.png)

```
/**
Beautifier
*/
```
